content/docs/concepts/examples.md

```diff
@@ -0,0 +1,40 @@
++++
+title = "Examples"
+description = "NSM use-case examples"
+weight = 6
+date = "2023-11-28"
++++
+
+## Examples
+A number of prepared examples of different ways to use nsm
+
+* [Basic examples](https://github.com/networkservicemesh/deployments-k8s/tree/release/v1.11.0/examples/basic) 
```

Comment:
I think we should use latest release in links: `v1.11.1`

Reply:
Thanks, fixed